core-java/src/main/java/eu/neverblink/jelly/core/RdfHandler.java

```diff
@@ -0,0 +1,85 @@
+package eu.neverblink.jelly.core;
+
+/**
+ * Interface for handling different types of RDF data structures that flow from the decoder.
+ *
+ * @param <TNode> The type of the nodes in the RDF data structure, as bound by library.
+ */
+public interface RdfHandler<TNode> {
+    /**
+     * Handle namespace definition.
+     * @param prefix The prefix of the namespace.
+     * @param namespace The namespace IRI, as represented by node in the RDF data structure.
+     */
+    default void handleNamespace(String prefix, TNode namespace) {
+        // No-op
+    }
+
+    /**
+     * Extension of the ProtoHandler interface to handle triples.
+     * @param <TNode> The type of the nodes in the RDF data structure, as bound by library.
+     */
+    interface TripleStatementHandler<TNode> extends RdfHandler<TNode> {
```

Comment:
Just TripleHandler, please...
QuadHandler
GraphHandler
AnyStatementHandler

Triples + Quads = statements in RDF